python/unityagents/communicator.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+from communicator_objects import UnityRLOutput, UnityRLInput,\
+    UnityOutput, UnityInput, AcademyParameters,\
+    UnityInitializationInput, UnityInitializationOutput
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger("unityagents")
+
+
+class Communicator(object):
+    def __init__(self, worker_id=0,
+                 base_port=5005):
+        """
+        Python side of the communication
+
+        :int base_port: Baseline port number to connect to Unity environment over. worker_id increments over this.
+        :int worker_id: Number to add to communication port (5005) [0]. Used for asynchronous agent scenarios.
+        """
+
+    def get_academy_parameters(self, python_parameters) -> AcademyParameters:
```

Comment:
Would it make sense to call this more generically get_parameters/get_environment_params? If you remove/rename the academy, you'll have to rename this api everywhere, especially since you mention the method is used to exchange initialization params between python and the **environment**

Reply:
True, but I think EnvironmentParameters will be used somewhere else (replace reset parameters in the long term). I think if we change the name here, we would have to change it also in the proto object and on the C# side. I imagine something more along the lines of environment_initialization_parameters.

Reply:
Love environment_initialization_parameters